lisa/operating_system.py

```diff
@@ -331,6 +343,11 @@ def _initialize_package_installation(self) -> None:
         # sub os can override it, but it's optional
         pass
 
+    def _get_package_information(
```

Comment:
The subclasses should implement this method. The super class (`Posix`) should implement get_package_information. When it needs the information call underlying `_get_package_information`.